lifecycle/src/test/scala/it/pagopa/interop/agreementprocess/lifecycle/AttributesRulesSpec.scala

```diff
@@ -1,40 +1,41 @@
 package it.pagopa.interop.agreementprocess.lifecycle
 
 import cats.implicits.catsSyntaxOptionId
-import it.pagopa.interop.agreementmanagement.client.model.Agreement
 import it.pagopa.interop.agreementprocess.lifecycle.AttributesRules._
-import it.pagopa.interop.tenantmanagement.client.model.Tenant
 import org.scalatest.matchers.should.Matchers._
 import org.scalatest.wordspec.AnyWordSpecLike
 
+import it.pagopa.interop.catalogmanagement.model.CatalogDescriptor
+import it.pagopa.interop.tenantmanagement.model.tenant.PersistentTenant
+import it.pagopa.interop.agreementmanagement.model.agreement.PersistentAgreement
+
 import java.time.OffsetDateTime
 import java.util.UUID
-import it.pagopa.interop.catalogmanagement.client.model.EServiceDescriptor
 
 class AttributesRulesSpec extends AnyWordSpecLike {
 
   "Certified attributes check" should {
-    "return true if all EService single attributes are satisfied" in {
+    "return true if all CatalogItem single attributes are satisfied" in {
```

Comment:
You can leave `EService` as conceptual model here. `CatalogItem` is a legacy name.
Same for all other test descriptions in this file.

```suggestion
    "return true if all EService single attributes are satisfied" in {
```